assets/src/stories-editor/components/rotatable-box/index.js

```diff
@@ -43,45 +48,45 @@ class RotatableBox extends Component {
 		const { keyCode } = e;
 
 		if ( ESCAPE === keyCode ) {
-			this.elementRef.classList.remove( 'is-rotating' );
-			this.elementRef.style.transform = `rotate(${ this.props.initialAngle }deg)`;
-
-			this.setState(
-				{
-					isRotating: false,
-					angle: this.props.initialAngle,
-				},
-				() => this.props.onRotateStop && this.props.onRotateStop( e, this.props.initialAngle )
-			);
+			elementRef.classList.remove( 'is-rotating' );
```

Comment:
Similarly, I'd do this in a `useEffect` utilising that `classList` has a toggle method:

```
useEffect( () => {
	elementRef.classList.toggle( 'is-rotating', isRotating );
}, [ isRotating ] );
```